backend/dal/Migrations/Initial/PostDeploy/01-ProjectStatus.sql

```diff
@@ -51,49 +51,57 @@ INSERT INTO dbo.[ProjectStatus] (
 ), (
     5
     , 5
+    , 'Approval Confirmation'
```

Comment:
This isn't correct based on the template provided by Christine and approved by Mikaela.

> Draft > Select Properties > Update Information > Required Documentation > Approval > Submitted

The user will need to confirm approval before the project can move from **Approval** to **Submitted**.  There is no requirement for an additional Project Status.

Reply:
I had a short discussion with @devinleighsmith, it is not clear where the Approval Confirmation step should be

Reply:
Ok, looks like it is in Review and Submit